packages/web/src/questionnaire/trackHelp.ts

```diff
@@ -46,7 +46,7 @@ export const help = {
       }
     },
     {
-      disabled: true,
+      disabled: false,
```

Comment:
todo: do not forget to rollback here and put `disabled: true` for this value. Otherwise a third option appears on this track